src/crypto-providers/trezorCryptoProvider.ts

```diff
@@ -221,41 +236,54 @@ const TrezorCryptoProvider: () => Promise<CryptoProvider> = async () => {
     })
   }
 
-  const prepareStakingKeyRegistrationCert = (
-    cert: _StakingKeyRegistrationCert,
+  const _prepareStakingKeyOrScriptCert = (
+    stakeCredentials: StakeCredentials,
     stakeSigningFiles: HwSigningData[],
-  ): TrezorTypes.CardanoCertificate => {
-    const path = findSigningPathForKeyHash(cert.pubKeyHash, stakeSigningFiles)
-    if (!path) throw Error(Errors.MissingSigningFileForCertificateError)
-    return {
-      type: TrezorTypes.CardanoCertificateType.STAKE_REGISTRATION,
-      path,
+  ): {path: string | number[], scriptHash?: string} => {
```

Comment:
The path here should be optional. However, it is also not optional in the Trezor Connect typescript definitions. I've updated it and will send you the updated package.

Reply:
I've made a Connect pre-release with the updated certificate type: https://github.com/vacuumlabs/connect/releases/tag/8.1.31-rc.1-multisig-extended